unicode_smilies/unicode_smilies.php

```diff
@@ -549,19 +351,24 @@ function unicode_smilies_smilies(&$a,&$b) {
 //	Smilies::add($b, ':deaf man:', '&#x1F9CF&#x200D&#x2642&#xFE0F');
 //	Smilies::add($b, ':deaf woman:', '&#x1F9CF&#x200D&#x2640&#xFE0F');
 	Smilies::add($b, ':person bowing:', '&#x1F647');
+	Smilies::add($b, ':bow:', '&#x1F647;');
 	Smilies::add($b, ':man bowing:', '&#x1F647&#x200D&#x2642&#xFE0F');
 	Smilies::add($b, ':woman bowing:', '&#x1F647&#x200D&#x2640&#xFE0F');
 	Smilies::add($b, ':person facepalming:', '&#x1F926');
+    Smilies::add($b, ':facepalm:', '&#x1F926;');
```

Comment:
Code standards: Please normalize the indentation.
```suggestion
	Smilies::add($b, ':facepalm:', '&#x1F926;');
```